tasks/update-code/git.yml

```diff
@@ -1,6 +1,30 @@
 ---
+
+- name: ANSISTRANO | GIT | Ensure GIT deployment key is up to date
+  copy:
+    src: "{{ ansistrano_git_identity_key_path }}"
+    dest: ".ssh/git_identity_key"
```

Comment:
We define it in our deployment playbook along with all the other variables for the deploy role, in that case the path is relative the the playbook itself (```./``` is the directory in which you have created your playbook).

I'll create a PR soon to make this clearer in the docs